configs/nvme-storage-configurator/daemonset.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: nvme-storage-configurator
+  labels:
+    app: nvme-storage-configurator
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      app: nvme-storage-configurator
+  template:
+    metadata:
+      labels:
+        app: nvme-storage-configurator
+    spec:
+      nodeSelector:
+        node-role.kubernetes.io/konflux-tenants: ""
+      hostPID: true
+      serviceAccount: nvme-storage-configurator
+      initContainers:
+        - name: configure
+          image: "quay.io/openshift/origin-network-tools"
+          command: ['/bin/bash', '-c', 'mkdir /cache && cp /scripts/configure.sh /cache && chmod +x /cache/configure.sh && /bin/bash /cache/configure.sh && exit 0']
```

Comment:
you don't need the `exit 0` at the end.

Reply:
what is the purpose of this cache dir anyway? can we just call `/bin/bash /scripts/configure.sh` ?

Reply:
I have shortened it.

`'/bin/bash', '-c', 'chmod +x /scripts/configure.sh && /bin/bash /cache/configure.sh'`

Reply:
fixed now.